lib/reactive_table.js

```diff
@@ -60,6 +60,13 @@ var updateHandle = function (set_context) {
         var onError = function (error) {
             console.log("ReactiveTable subscription error: " + error);
         };
+        if(context.serverArgs) {
+          if(_.isFunction(context.serverArgs)) {
+            var serverArgs = context.serverArgs.call(this);
```

Comment:
To me it would make most sense for the `this` inside the serverArgs function to be the same this as the `this` at the level where one calls `{{> reactiveTable ...}}`. (The same as calling `Template.parentData()` inside the `context` helper for `Template.reactiveTable`.)